rke/templates/templates.go

```diff
@@ -165,12 +165,12 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			// New ingress template introduced for 1.15.12-rancher1-1, 1.16.10-rancher1-1, 1.17.6-rancher1-1
 			">=1.15.12-rancher1-1 <1.16.1-rancher1-1": nginxIngressV11512,
 			">=1.16.10-rancher1-1 <1.17.0-rancher1-1": nginxIngressV11512,
-			">=1.17.6-rancher1-1 <1.19.15-rancher1-3": nginxIngressV11512,
-			">=1.19.15-rancher1-3 <1.20.0-rancher0":   nginxIngressv104,
-			">=1.20.0-rancher0 <1.20.11-rancher1-3":   nginxIngressV11512,
-			">=1.20.11-rancher1-3 <1.21.0-rancher0":   nginxIngressv104,
-			">=1.21.0-rancher0 <1.21.5-rancher1-2":    nginxIngressv0481,
-			">=1.21.5-rancher1-2":                     nginxIngressv104,
+			">=1.17.6-rancher1-1 <1.19.16-rancher1-1": nginxIngressV11512,
```

Comment:
Just to call out that we will only update the nginx version on each latest version of 1.19, 1.20 and 1.21, correct?

Reply:
yes, that's correct.